maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlConfig.kt

```diff
@@ -16,6 +18,8 @@ data class YamlConfig(
     val initFlow: YamlInitFlowUnion?,
     val tags: List<String>? = emptyList(),
     val env: Map<String, String> = emptyMap(),
+    val onFlowStart: YamlOnFlowStart?,
+    val onFlowComplete: YamlOnFlowComplete?,
```

Comment:
What was the point to prefer this over before/after again? I remember some point over extensibility but just want to ensure I understand that correct

Reply:
for the case of extensibility, i.e. if someone will need onFlowFailed / onFlowSucceeded hooks in the future